StatsdClient/Statsd.cs

```diff
@@ -181,6 +201,25 @@ private void InitialiseInternal(Func<IOutputChannel> createOutputChannel, string
         {
             if (value < 0)
             {
+                throw new ArgumentOutOfRangeException();
```

Comment:
I don't think this should throw an exception, only log it and return like the others. Stats client really shouldn't be blowing up.